README.md

```diff
@@ -1,19 +1,27 @@
 # FFC Payment Batch Processor
 
-FFC service to process Siti Agri batch files and send individual transactions as messages.
+## Description
 
-## Prerequisites
+FFC microservice to process Siti Agri batch files and send individual transactions as messages.
+Receives batch files from FFC Pay Batch validator and sends payment requests on to FFC Pay Enrichment.
```

Comment:
Suggest if we reference other repositories, we should provide a link to them to help readers?

eg `Receives batch files from [batch validator](https://github.com/DEFRA/ffc-pay-batch-validator)`

However! I think we need to think about how we can effectively describe the role of this service without tightly coupling it to the others.  ie if we replace batch validator then this repo needs updating too, if we add another input then it's out of date.  So maybe we shouldn't be too detailed over other microservices that exist that are part of the wider ecosystem.

I'd instead focus on this repo reading payment files from Azure Blob storage, validating the payment requests and publishing validated payment requests.  What service or services that consume those messages aren't a concern of this repository.

Maybe we should have a link to a diagram of the end to end service so we only need to maintain that in once place?  Perhaps in the `ffc-pay-core` repo or confluence or `ffc-diagrams`.

What do you think?